meta-raspberrypi-mbl/conf/machine/raspberrypi3-mbl.conf

```diff
@@ -83,3 +83,13 @@ IMAGE_BOOT_FILES += "bcm2835-bootfiles/* \
                     '${KERNEL_IMAGETYPE} u-boot-nodtb.bin;${SDIMG_KERNELIMAGE}', \
                     '${KERNEL_IMAGETYPE};${SDIMG_KERNELIMAGE}', d)} \
                  "
+
+##############################################################################
+# Partition layout config
+##############################################################################
+# BL2 is stored in a FAT partition on rpi3, not in an area of raw storage
+MBL_WKS_SKIP_BOOTLOADER1 = "1"
+
+# Put the BL3 FIT at an offset of 1024KiB
+MBL_WKS_BOOTLOADER2_ALIGN_KiB = "1024"
```

Comment:
Can you please follow the same order that we have for the other machines?
```
MBL_WKS_BOOTLOADER2_FILENAME = "fip2.bin"
MBL_WKS_BOOTLOADER2_ALIGN_KiB = "1024"
```

Reply:
https://github.com/ARMmbed/meta-mbl/pull/610/commits/0b58a217666db5253330de6a9230fea75aacd148